app/controllers/jobs_controller.rb

```diff
@@ -39,6 +39,15 @@ def run
     end
   end
 
+  def retry_queued
+    @jobs = Delayed::Job.where('failed_at IS NULL AND attempts > 0 AND locked_at IS NULL ').update_all(run_at: Time.now, attempts: 0)
```

Comment:
Another thought.  Should we be setting `attempts: 0` like you have, or should we leave it alone and just set `run_at` so they run now without resetting their try count?